README.md

```diff
@@ -27,10 +27,14 @@
 PyAEDT is a Python library that interacts directly with the API for
 Ansys Electronics Desktop (AEDT) to make scripting simpler. The architecture
 for PyAEDT can be reused for all AEDT 3D products (HFSS, Icepak, Maxwell 3D,
-and Q3D Extractor), 2D tools, and Ansys Mechanical. PyAEDT also provides
+and Q3D Extractor), 2D tools, and Ansys Mechanical inside AEDT. PyAEDT also provides
 support for circuit tools like Nexxim and system simulation tools like
 Twin Builder. Finally, PyAEDT provides scripting capabilities in Ansys layout
-tools like HFSS 3D Layout and EDB. The PyAEDT class and method structures
+tools like HFSS 3D Layout. The Ansys Electronics Database 
+([EDB](https://edb.docs.pyansys.com/version/stable/)) is included
+with PyAEDT as a dependency and is recommended for any automated manipulation and 
+setup of layout data for PCBs, electronic packages and integrated circuits. 

```

Comment:
```suggestion
setup of layout data for PCBs, electronic packages, and integrated circuits. 

```
We use the Oxford comma, which is to always place a comma before the "and" or the "or" in a list of three or more items.